pyproject.toml

```diff
@@ -46,7 +46,7 @@ postgres = ["sqlalchemy", "psycopg2-binary"]
 databricks = ["databricks-sql-connector"]
 
 [tool.poetry.group.sql.dependencies]
-great-expectations = { version = "^0.18.19"}
+great-expectations = { version = ">=0.18.19,<1.0"}
```

Comment:
Does this also need to be changed in the `[tool.poetry.dependencies]` section @Kilo59?

Reply:
Yes! Good catch!
I thought that's what I was doing 😆